openvpn/start.sh

```diff
@@ -11,6 +11,15 @@ echo "Using OpenVPN provider: $OPENVPN_PROVIDER"
 
 if [ ! -z "$OPENVPN_CONFIG" ]
 then
+	n=$(echo "$OPENVPN_CONFIG" | wc -w)
```

Comment:
This whole block could be something like the following:
```
OPENVPN_CONFIG_ARRAY=( ${OPENVPN_CONFIG//,/ } )
if (( ${#OPENVPN_CONFIG_ARRAY[@]} > 1 )); then
     OPENVPN_CONFIG_RANDOM=$((RANDOM%${#OPENVPN_CONFIG_ARRAY[@]}))
     echo "${#OPENVPN_CONFIG_ARRAY[@]} servers found in OPENVPN_CONFIG, ${OPENVPN_CONFIG_ARRAY[${OPENVPN_CONFIG_RANDOM}]} chosen randomly"
     OPENVPN_CONFIG="${OPENVPN_CONFIG_ARRAY[${OPENVPN_CONFIG_RANDOM}]}"
 fi
```

Which removes all the calls to external applications (wc, awk, potentially echo, and probably [ and ]).

Instead of space separation, this block uses commas.

Reply:
Thanks alron,

This is all about coding style and code readibility.

When I read haugene code, he use tr in place of $(//), [ in place  of [[, ...
I consider my duty to add code in the author coding style, so he can maintain it easily.

IMHO, There is no reason to optimize 2 commands when they are not executed a zillion times in a loop or where performance is not an issue.
Using sed or ${//}, cut or ${#%%} is useful when performance is an issue, this is not the case here, and this is done at the expense of code readibility and portability.

I can code both styles, it's finnaly a matter of taste.

Reply:
Unfortunately, as some of .ovpn files for PIA use spaces in their names, it's not about coding style. IMHO, the delimiter shouldn't be a character eligible for a filename.
See #445.

Reply:
@vSh1ny #440 includes the comma delimiter block I suggested above, along with other firewall updates.

Reply:
@alron Whoops, didn't see that. Great PR, waiting for the merge. Thanks.